dts/xtensa/esp32.dtsi

```diff
@@ -0,0 +1,43 @@
+#include "skeleton.dtsi"
+
+/ {
+	cpus {
+		#address-cells = <1>;
+		#size-cells = <0>;
+
+		cpu@0 {
```

Comment:
Do we support interrupts on xtensa?  If so we need interrupt controller node

Reply:
No esp32 doesn't support.